src/webhooks/webhooks.service.ts

```diff
@@ -102,14 +104,20 @@ export class WebhooksService {
 
     try {
       if (story.content?.component === 'Course') {
-        await this.courseRepository
+        const { identifiers } = await this.courseRepository
           .createQueryBuilder('course')
           .insert()
           .into(CourseEntity)
           .values(createCourseObject)
           .onConflict(`("storyblokId") DO UPDATE SET "status" = '${action}'`)
```

Comment:
we also want to update the name and slug if they've changed

if we used `this.courseRepository.save` instead, and didnt specify the fields, the whole record would be updated the match the `createCourseObject` above? i think thats what we want